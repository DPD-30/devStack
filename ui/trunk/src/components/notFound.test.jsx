import React from 'react'
import { renderToString } from 'react-dom/server'


import NotFound from './notFound.jsx'

jest.unmock('./notFound.jsx')


test('Not found message', () => {
	const comp = renderToString(<NotFound />)

	expect(comp).toMatchSnapshot()
})
