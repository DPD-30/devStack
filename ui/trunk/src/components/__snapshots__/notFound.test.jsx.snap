// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found message 1`] = `
<div data-reactroot>
  Sorry, that page could not be found
</div>
`;
